packages/apps/stock/src/components/templates/customerDelivery/CustomerDeliveryLineCard/CustomerDeliveryLineCard.tsx

```diff
@@ -75,15 +79,25 @@ const CustomerDeliveryLineCard = ({
     return Colors.successColor.background;
   }, [Colors, askedQty, pickedQty]);
 
-  const borderStyle = useMemo(() => {
-    return getStyles(borderColor)?.border;
+  const styles = useMemo(() => {
+    return getStyles(borderColor);
   }, [borderColor]);
 
+  useEffect(() => {
+    checkQuantityApi({stockMoveLineId})
+      .then(response => {
+        if (response?.data?.object) {
+          setCheckQtyObject(response?.data?.object);
+        }
+      })
+      .catch(() => setCheckQtyObject(null));
+  }, [stockMoveLineId]);
```

Comment:
Can we imagine to create a hook to manage the process, as it is the same thing on multiple screen/components